Chat/appsettings.json

```diff
@@ -1,11 +1,29 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Trace",
-            "System": "Information",
-            "Microsoft": "Information"
-        }
-    },
-    "AllowedHosts": "*",
-    "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING"
+  "Logging": {
+    "LogLevel": {
+      "Default": "Trace",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ResourceConnectionString": "endpoint=https://whatchpartyhackaton.communication.azure.com/;accesskey=kP3OcxizuwOqKHx8qzpeUJmHNQ2e8E+6m64h5YgcuFjai12YvaZGkPgifxvAGfjBSHD7UuFqPRd3DNFQl0W7nA==",
+  "LiveStreaming": {
+    "_comment": "For security reasons, please do not check in this file to source control.",
```

Comment:
@JianpingChen you might want to sanitize your secrets after this, for the AAD client etc